ios/NotifeeCore/NotifeeCore+UNUserNotificationCenter.m

```diff
@@ -133,10 +133,22 @@ - (void)userNotificationCenter:(UNUserNotificationCenter *)center
 - (void)userNotificationCenter:(UNUserNotificationCenter *)center
     didReceiveNotificationResponse:(UNNotificationResponse *)response
              withCompletionHandler:(void (^)(void))completionHandler {
-  NSDictionary *notifeeNotification =
-      response.notification.request.content.userInfo[kNotifeeUserInfoNotification];
+  NSMutableDictionary *notifeeNotification =
+      [response.notification.request.content.userInfo[kNotifeeUserInfoNotification] mutableCopy];
+
+  NSLog(@"notifee didReceiveNotificationResponse notifee %@", notifeeNotification);
```

Comment:
does NSLog turn off in non debug build schemes?

Reply:
mmm 🤔  good spot. must have left this is by mistake so have now removed. 

Although we do have this in the codebase but it's normally on failures so the developer can see in their logs.

Reply:
Yeah in an otherwise quiet-failure mode I'll advocate for these sometimes, but I spotted this on the happy path and thought :thinking: